libs/dal/src/repositories/subscriber/subscriber.schema.ts

```diff
@@ -177,13 +177,9 @@ subscriberSchema.index({
  * We can not add `deleted` field to the index the client wont be able to delete twice subscriber with the same subscriberId.
  */
 subscriberSchema.index(
-  {
-    subscriberId: 1,
-    _environmentId: 1,
-  },
-  { unique: true }
+  { _environmentId: 1, subscriberId: 1 },
```

Comment:
as we talked before, lets keep the subscriberId first as he has higher cardinality. 

ref: 
https://stackoverflow.com/questions/33545339/how-does-the-order-of-compound-indexes-matter-in-mongodb-performance-wise

https://books.google.co.il/books?id=uGUKiNkKRJ0C&pg=PA98&lpg=PA98&dq=mongodb+the+definitive+guide+index+cardinality&source=bl&ots=h8mzPdcTrl&sig=xJ40rE2qjoSy9RQrvxLnNDx6KS8&hl=en&sa=X&redir_esc=y#v=onepage&q=mongodb%20the%20definitive%20guide%20index%20cardinality&f=false